src/tools/Scoop/Get-Sc9InstallData.ps1

```diff
@@ -0,0 +1,45 @@
+<#
+.SYNOPSIS
+Installs all required packages and returns the paths required for a Sitecore 9 installation.
+
+.DESCRIPTION
```

Comment:
if this Cmdlet is supposed to use as "external", so called not only "private", I would think about describing what the return object looks like, in order to make this understandable for further usages